test/Serilog.Tests/Core/LoggerTests.cs

```diff
@@ -137,7 +137,7 @@ public void TheNoneLoggerIsAConstant()
     [Fact]
     public void TheNoneLoggerIsSingleton()
     {
-        lock (new object())
+        lock (this)
```

Comment:
I don't think any other code locks `this` so there's probably still no synchronization going on here; I'll merge this change as I think it shows the intent of the old code better, but we'll need to configure xUnit to serialize test execution around the static `Log` class separately, if there's any code outside of `LoggerTests` that interacts with it.